src/ontology/edits/oeo-physical.omn

```diff
@@ -5286,9 +5296,14 @@ Class: OEO_00050011
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "An energy unit which is equal to 10^9 watt-hours.",
-        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/606
+        <http://purl.obolibrary.org/obo/IAO_0000233> "add eurostat synonym:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/920
+pull: https://github.com/OpenEnergyPlatform/ontology/pull/924
+
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/606
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/608",
         <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> "GWh",
+        <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> "gigawatt hours",
```

Comment:
I thought we decided to only use `alternative term`. Maybe `hasExactSynonym` is an artifact?

Reply:
You are right. Just changed it.